project/ottersome/README.md

```diff
@@ -0,0 +1,186 @@
+# RacImgLib(RIL): General Purpose Image Processing Framework
+
+This is a general purpose framework for image processing tasks. Its aim for the
+course will be humble in purpose but its lifetime will be longer than the
+course. It will start by with the aim of solving 3 main tasks related to image
+processing. This will be stated concretely and tackled generally. Once the
+course is over my aim is to use it for my own photography needs. The tools will 
+be more on the image editing/post-production side rather than something like
+openCV. A little disclaimer will be made: Due to restrictions in time I will
+most likely be able to implement only one or two of these features, but I state
+three in case time allows to cover them all. If I find that one of them doesn't
+interest me or is too lengthy then I might switch them as well.
+
+# Basic Information
+
+[Repository can be found here](https://github.com/ottersome/RIL).
+
+# Problem to Solve
+
+This is mostly a utility that solves problems of incovenience for myself. As
+I want a longer lasting project than a single semester I went with a broader
+subject.
+This is of use to anyone or any industry doing image processing. 
+For the sake of obtaining a sense of completion for the course, the three main goals
+will be chosen. Their inclusion is simply because they are a mix of previously acquired knowledge
+and experience I have been meaning to acquire
+
+1. [JPG Compression](https://people.cs.rutgers.edu/~elgammal/classes/cs334/slide9_short.pdf)
+   This uses a uses lossy compression through transform coding(DCT).
+   If time allows, I would also like to implement variations to see if there any
+   benefits to them.
+2. Kernel Application: A simple way of specifying your kernels and seeing their
```

Comment:
By kernel, do you mean something like a gaussian blur?

Reply:
Yes, Gaussian, Sobel, the like...